.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,55 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
-
-You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:
+<!-- READ THIS FIRST:
+- Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
+- Do not report issues for components if you are using custom components: files in <config-dir>/custom_components
+- Frontend (user interface of Home Assistant in your browser) related issues have to be reported here: https://github.com/home-assistant/home-assistant-polymer/issues
+- This is for bugs only. Feature and enhancement requests should go in our community forum: https://community.home-assistant.io/c/feature-requests
+- Answer all question as detailed as possible, write the information inside the backticks. Do not delete any text from this template!
+
+-->
+**1. Home Assistant version:**
+```
 
-**Home Assistant release (`hass --version`):**
+```
+<!--
+- Frontend -> developer tools -> info
+- Or use this command: hass --version
+-->
 
+**2.a) I run Hass.io or the Docker image**:
+```
 
-**Python release (`python3 --version`):**
+```
+<!--
+- Yes / No
+-->
 
+**2.b) ...No, I run an installation with this Python version:**
+```
 
-**Component/platform:**
+```
+<!--
+- Minimum supported version is Python 3.5.3
+- Use this command: python3 --version
+-->
 
+**3. Component/platform:**
+```
 
-**Description of problem:**
+```
+<!--
+- example: components/updater.py
+-->
+**4. Description of problem:**
+```
 
```

Comment:
Problem with writing description inside backticks is that there's no soft wrapping inside code blocks:

```
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```